tools/gn

```diff
@@ -216,7 +216,7 @@ def main(argv):
   if sys.platform.startswith(('cygwin', 'win')):
     subdir = 'win'
   elif sys.platform == 'darwin':
-    subdir = 'mac'
+    subdir = 'mac-x64'
```

Comment:
I can’t remember if the clang-format presubmit references this but worth checking.

Reply:
Where are the presubmits?

Reply:
`travis/[...]`, though looks like it should be fine.